plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -345,6 +345,40 @@ public Object[][] predicatePushdownProvider()
         };
     }
 
+    @Test(dataProvider = "floatingPointPredicatePushdownProvider")
```

Comment:
Note that we're migrating `dataProvider` to the helper method. See https://github.com/trinodb/trino/commit/e23214d8eeb66c090fd713ad12b1ffae12f5f7c4

Also, we can split into `testPredicatePushdownReal` and `testPredicatePushdownDouble`.